packages/api-db-mongodb/package.json

```diff
@@ -35,7 +35,9 @@
   "dependencies": {
     "bcrypt": "^5.0.0",
     "mongodb": "^3.5.4",
-    "nanoid": "^2.1.11"
+    "nanoid": "^2.1.11",
+    "node-input-validator": "^4.5.0",
+    "normalize-email": "^1.1.1"
```

Comment:
This library hasn't been updated in years.  If the user explicitly sets a different email, we probably should let them.

I for example use email+company/type to keep track (using filters, labels etc) of emails.